functorch/_src/aot_autograd.py

```diff
@@ -387,6 +387,12 @@ def fake_fn(primals, tangents):
     with torch.no_grad():
         with track_graph_compiling("joint"):
             fw_module, bw_module = aot_config.partition_fn(fx_g, joint_inputs)
+            fw_outs = [n for n in fw_module.graph.nodes if n.op  == "output"][0].args[0]
+            # we only need to bookkeep the symints that are saved for bw, not any symints
+            # the user forward might have returned in its own output
+            fw_outs = fw_outs[_num_outs:]
+            symint_outs = [n for n in fw_outs if "aten.sym_size" == str(n.target)]
```

Comment:
Should probably just use if `n.target == aten.sym_size.default`

Reply:
Hmmm if you have a graph like this:
```
def fwd(...):
    ....
    a = intermediate.shape
    b = operator.getitem(a, 0)
    c = b + b
    return c
```

Then won't the output node's `n.target` look something like `"add"` (or some other symint operation)?

Today during tracing, we already store fake tensor info directly on the fx nodes - I'm wondering if it would be more robust to also store symints (or at least a flag indicating that the node matches to a symint), so you can just check that here?
```
symint_outs = [n for n in fw_outs if n.meta['is_symint']]
```

Reply:
thanks @bdhirsh - i'll add this test case and fix.